csaf_2.1/prose/edit/src/tests-01-mndtr-11-cwe.md

```diff
@@ -1,20 +1,24 @@
 ### CWE
 
-It MUST be tested that given CWE exists and is valid.
+It MUST be tested that given CWE exists and is valid in the version provided.
+Any `id` that refers to a CWE Category or View MUST fail the test.
 
 The relevant path for this test is:
 
 ```
-    /vulnerabilities[]/cwe
+    /vulnerabilities[]/cwes[]
```

Comment:
Cf. my feedback on the CWE container as singular.

Reply:
see https://github.com/oasis-tcs/csaf/pull/738#discussion_r1615733580